### PR TITLE
[WIP] Add Profile Link for each Collaborator

### PR DESCRIPTION
**Pull request by gep13**
What do you think about this addition?

image

It is still a work in progress, but just wanted to get a feel for whether you thought it was a good idea or not.

### CODE REVIEW

**Comment by Odonno** [comment]
Yeah, good idea !

**Comment by gep13** [comment]
>@Odonno said...
Yeah, good idea !

Cool, in which case I will finish up the implementation.

Any ideas why the Flyout is appearing as black on this screen, when it is white in the Room screen?

**Comment by Odonno** [comment]
@gep13 Not really, no.

**Comment by gep13** [comment]
@Odonno I see now why this was required:

```
#if WINDOWS_PHONE_APP
            await Windows.System.Launcher.LaunchUriAsync(new Uri($"http://www.github.com/{userName}"));
#endif
```

It is due to the fact that `Windows.System.Launcher` is only available in the Windows Phone Application, and not in Gitter.W10 or Gitter.Windows:

image

As a result, I will have to leave the compiler directive in place, unless you have an alternative suggestion

**Comment by Odonno** [comment]
@gep13 Ok, well, I forgot that. I have no alternative, it seems taht #if is perfect for now.

**Comment by gep13** [comment]
ok, so we can leave it as it is just now.

Also, I can't figure out why the MenuFlyout has a background colour of black, when in the other screen it is white.  I can't think why that would be.  

I have now tested the Context Menu, and it correctly navigates to the profile page of the collaborator.

File: Gitter/Gitter/Gitter.WindowsPhone/Views/AboutPage.xaml [hunk 1]
```diff
@@ -92,16 +95,32 @@
                                     <ColumnDefinition Width="60" />
                                     <ColumnDefinition />
                                 </Grid.ColumnDefinitions>
-                                
-                                <Image Grid.Column="0" 
-                                       Source="{Binding Path=Image}" />
+
+                                <Grid Grid.Column="0">
```
Comment (Odonno) [inline Gitter/Gitter/Gitter.WindowsPhone/Views/AboutPage.xaml]: Add `RequestedTheme="Dark"`to make flyout background white...

Comment (gep13) [inline Gitter/Gitter/Gitter.WindowsPhone/Views/AboutPage.xaml]: The `MenuFlyout` doesn't have the property, and adding it to the `MenuFlyoutItem` which does have that property, doesn't seem to have any effect.

Comment (Odonno) [inline Gitter/Gitter/Gitter.WindowsPhone/Views/AboutPage.xaml]: Sorry, add it to the near container, so the `Grid`.

Comment (gep13) [inline Gitter/Gitter/Gitter.WindowsPhone/Views/AboutPage.xaml]: Ah gotcha, okay that works, and I can now see that this is the same on the ChatMessageControl, so it all makes sense now.  I have pushed the changes, so this should be ready for merging in now.

**Comment by Odonno** [comment]
Yeah ! Good job on that !

**Pull request merged**